### DISCUSSION TITLE
关于鉴权access_token相关的兼容性问题

### DISCUSSION THREAD

**Discussion by awp9835** [howmanybots/onebot#130] [想法 / 建议]
有的access_token会带来兼容问题。 
1.  未设置access_token时，视为空字符串还是无access_token；
2.  access_token设置为""空字符串时，如何处理；
3.  如果access_token的两端含有空格字符，某些情况下会因为trim产生不兼容，是否应该规范trim的行为

**Comment by crazywhalecc** [comment]
access_token 偏向于一个鉴权的约定串，未设置时视为无 token。

空字符串时视为无 token。

access_token 在验证时无需 trim，但是在放置在 Header、GET 等位置时可能需要使用 trim，这时以不同语言自己的实现为准，不做规定。

**Comment by stdrc** [comment]
感觉可以在标准里面明确一下

**Reply by stdrc** [reply]
OK 已经明确了
